__DEFINES/setup.dm

```diff
@@ -720,21 +720,8 @@ SEE_PIXELS	256
 
 #define CUT 		"cut"
```

Comment:
It seems like you should either move these to wherever the other damage defines are, or remove those ones instead